zebra/zapi_msg.c

```diff
@@ -2360,6 +2360,20 @@ static inline void zread_rule(ZAPI_HANDLER_ARGS)
 		if (zpr.rule.filter.fwmark)
 			zpr.rule.filter.filter_bm |= PBR_FILTER_FWMARK;
 
+		if (!(zpr.rule.filter.src_ip.family == AF_INET
+		      || zpr.rule.filter.src_ip.family == AF_INET6)) {
+			zlog_debug("Unsupported PBR source IP family: %s\n",
```

Comment:
not zlog_warn in this case?

Reply:
well at the very least it needs to be debug guarded :)

Reply:
Hmm, does it? Seems like spamming the log in this case is something you'd want so we get bug reports. I think I'll change it to warn and leave it unguarded, sound good?

Reply:
@qlyoung That sounds good to me.